apiary.apib

```diff
@@ -1178,6 +1199,8 @@ There are a lot of filtering capabilities for this report. Most of the columns r
         - href: https://affiliate.api.diablomedia.com/v1/campaigns/1/creatives_export?publisher_id=10000 (string)
     - suppression
         - href: https://affiliate.api.diablomedia.com/v1/campaigns/1/suppression?publisher_id=10000 (string)
+    - optizmoaccess
+        - href: https://affiliate.api.diablomedia.com/v1/campaigns/1/optizmoaccess?publisher_id=10000 (string) ~ If available
```

Comment:
This `~ If available` doesn't render in Apiary, was this a syntax you found for an optional property?

Reply:
What looked like a ~ was a -, though I was just trying to add a "comment" there.  I'll see if there's a proper way to flag an optional property

Reply:
Hmm, nullable might apply here?  

"nullable - instance of this type Value MAY be unset (e.g. null or nil)."